### ISSUE TITLE
iPadOS scrolling is impossible

### ISSUE THREAD

**Issue by educatedspice99** [marcusolsson/obsidian-projects#727]
### What happened?

Multiple views in a project on iPadOS cannot be scrolled by touch. When touched, they only move.

### What did you expect to happen?

Expected behavior is scrolling by touch the multiple views.

### How can we reproduce it (as minimally and precisely as possible)?

Create multiple views on iPadOS and try to scroll through them.

### Plugin version

1.17.0

### Obsidian version

1.4.16

### OS

Other

**Comment by Acylation** [comment]
Yeah we should deal with touches more carefully. Add an interval before triggering dnd would be natural. So we can scroll directly, and sort with a long press.

I will also do some surveys on vibration apis to give users instant feedbacks on toggling scroll / sort.
